Docs/Configuration.tex

```diff
@@ -3888,6 +3889,12 @@ \subsection{Debug Properties}\label{miscdebugprops}
   avoid frequent use of this option when dealing with flash drives as large I/O
   amounts may speed up memory wear and render the flash drive unusable quicker.
 
+  \textbf{Warning}: It is possible to enable fast file logging, which works as indicated
```

Comment:
Better change it to `Warning 2` to match our other `Note`s.

Reply:
Will do